sycl/test-e2e/Matrix/get_coordinate_ops.cpp

```diff
@@ -12,6 +12,9 @@
 // REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 30049
 
+// XFAIL: cpu
+// XFAIL-TRACKER: https://github.com/intel/llvm/issues/17079
```

Comment:
Please change the tracker to CMPLRTST-26750
The issue is fixed but we need to wait for OCL CPU 2025.1 uplift to SYCLOS
The issue should not be there in xmain so please make sure this does not get replicated in xmain by cherry picking this to xmain and add a custom comment

Reply:
You can't be asking upstream open-source project to use some obscure internal bug identification. We do allow such references to avoid the needless creation of "mirror" GH issue in this project, but in this instance such GH issue already exists (and it had some history of investigation there already)!

Whatever else follows is caused by the Matrix team not working proactively to implement this markup and should be followed up by them.